lib/mongoid/criteria/queryable/selectable.rb

```diff
@@ -516,7 +517,7 @@ def nin(condition)
         # @example Add the $nor selection.
         #   selectable.nor(field: 1, field: 2)
         #
-        # @param [ Array<Hash | Criteria> ] criteria Multiple key/value pair
+        # @param [ [ Hash | Criteria ]... ] *criteria Multiple key/value pair
```

Comment:
This should retain `Array<>` to indicate the acceptance of arrays.

Reply:
Fixed